ckanext/datastore/writer.py

```diff
@@ -161,9 +161,20 @@ def writerow(self, row):
             root.attrib[u'_id'] = unicode(row[0])
             row = row[1:]
         for k, v in zip(self.columns, row):
+            k = self.getXmlElement(k)
+            if unicode(k).isnumeric():
+                k = u'_' + unicode(k)
             if v is None:
                 SubElement(root, k).attrib[u'xsi:nil'] = u'true'
                 continue
             SubElement(root, k).text = unicode(v)
         ElementTree(root).write(self.response, encoding=u'utf-8')
         self.response.write(b'\n')
+
+    def getXmlElement(self, element_name):
```

Comment:
`snake_case` is the convention for methods and functions. I'd also move this outside of the class and make it a simple function because `self` is not used anywhere in the body